country-dashboard/src/App.tsx

```diff
@@ -1,28 +1,61 @@
-import logo from './logo.svg';
-import './App.css';
-import countries from './data/countries.json';
+import "./App.css";
+import countries from "./data/countries.json";
+import Card from "./components/Card";
 
 export const App = () => {
   // countries holds a JSON array of the countries and their data which you can use
-  console.log(countries);
+
+  const countryBool = !countries || countries.length === 0;
+
+ 
+  const cardui = countries.map((item, index) => {
```

Comment:
While technically correct, it isn't great practice to declare components inside components (except for a few special cases). It messes with react's rendering sometimes, if the countries value is changed, this doesn't always get picked up if it's declared as another component. It's better to just do {countries.map(...)} inside the `return()` of this component.